quickstarts/microsoft.desktopvirtualization/avd-backplane-with-network-and-storage-and-monitoring/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
this is a subscription scoped deployment so needs to be manually validated - set the validationType property to manual and add valid param values (as appropriate